From 5e374136992be4189a995b9fe7e5dc7cf243baf7 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 21 Jun 2024 14:27:49 +0200
Subject: [PATCH 0932/1695] drivers: stm32_hsem: use generic CIDCFGR macros

Use _CIDCFGR_SCID_SHIFT instead of SCID_SHIFT.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 50d547084f92b0c28ef4456e20c4fb8633b2fdf4 drivers: stm32_hsem: use generic CIDCFGR macros]
Change-Id: I00f20eba334752cc3160f473886480e68932e560
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/389466
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/drivers/stm32_hsem.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/drivers/stm32_hsem.c b/core/drivers/stm32_hsem.c
index 88a36526e..f19cbb56d 100644
--- a/core/drivers/stm32_hsem.c
+++ b/core/drivers/stm32_hsem.c
@@ -233,7 +233,7 @@ static TEE_Result parse_dt(const void *fdt, int node)
 		unsigned int pos = fdt32_to_cpu(cuint[i * 2]) - 1;
 		unsigned int cid_value = fdt32_to_cpu(cuint[(i * 2) + 1]);
 
-		hsem_d->rif_proc_conf[pos] = cid_value << SCID_SHIFT;
+		hsem_d->rif_proc_conf[pos] = cid_value << _CIDCFGR_SCID_SHIFT;
 	}
 
 	return TEE_SUCCESS;
-- 
2.47.2

